ui/app/models/totp-key.js

```diff
@@ -57,8 +74,15 @@ export default class TotpKeyModel extends Model {
   })
   backend;
 
-  @attr('string') name;
-  @attr('string') accountName;
+  @attr('string', {
+    subText: 'Specifies the name for this key.',
+  })
+  name;
+
+  @attr('string', {
+    subText: `The name of the account associated with the key.`,
```

Comment:
The designs may not include this, but if a parameter is required, it's good to mention this in the subtext. We want to help users as much as possible before submitting a form. 
```suggestion
    subText: `The name of the account associated with the key. Required for keys generated by Vault.`,
```